src/mcp/tools/base.py

```
@@ -10,126 +13,93 @@
+    def _check_references(self, name: str) -> list[str]: ...
 
-        Args:
-            name: Name of the component to remove.
-        """
-        ...
+    def _iter_entries(self):
+        for object_type in self.object_types:
+            try:
+                items = self.state.idf.all_of_type(object_type)
+            except Exception:
+                continue
```

Ruff (0.16.0)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)